### ISSUE TITLE
Clarify whether aria-activedescendant may have a null value

### ISSUE THREAD

**Issue by brennanyoung** [w3c/aria#1970]
The spec for `aria-activedescendant` offers no guidance for what happens when there is no descendant (or owned) element to focus.

This may happen (e.g.) with a combobox under at least two distinct conditions:

- User has not yet entered anything in the input field
- User has entered a string which is not matched

Spec at https://www.w3.org/TR/wai-aria-1.2 **does not** recommend removing the attribute if it points at "nothing" (as it does with aria-errormessage) - this might imply that keeping it in the DOM (e,g, pointing at an empty string) is completely fine, but this is not very obvious. I'd like this clarified unambiguously.

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)?
Apparently yes.

**Comment by spectranaut** [comment]
@mpaiva maybe this can be worked on with aria-controls?

**Comment by mpaiva** [comment]
I agree @spectranaut - we should add that to the language.

**Comment by MarioBatusic** [comment]
Agree. An other example is an empty listbox. We use for example listboxes for object lists and these can be empty. After creating one object list, it is always empty. So you have to put the focus on it and at these circumstances the focus stys exaclty here. The aria-activedescendant can only be "none" at this time.

**Comment by mpaiva** [comment]
I am not sure if anyone has already proposed a way to keep the parent element informed of its descendent existence, but I am thinking that a new attribute could be of help for these dynamic elements. I'd be willing to write a proposal for `aria-pending`. 

`aria-pending`: This attribute could be used to indicate that a descendant element is pending or in the process of being mounted or created. It would inform parent elements, such as `aria-controls` or `aria-owns`, and assistive technologies that the descendant is not currently available but is expected to be added or become accessible in the future.

The `aria-pending` attribute could have Boolean values such as “true” or “false” to indicate the pending state of the descendant. When the descendant is finally mounted or available, the attribute could be updated to reflect the change.

@spectranaut - would you think this is viable?

**Comment by MarioBatusic** [comment]
I think that aria-busy put on the parent element has already the same functionality?

**Comment by mpaiva** [comment]
> I think that aria-busy put on the parent element has already the same functionality?

Indeed. Thanks @MarioBatusic – this helpful an I can add this approach in the proposed language changes for the `aria-controls` attribute.
